topics/assembly/tutorials/get-started-genome-assembly/slides.html

```diff
@@ -0,0 +1,394 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "An introduction to get started in genome assembly and annotation"
+questions:
+  - "What are the guidelines before starting a Genome Assembly and Annotation project?"
+contributors:
+  - abretaud
+  - alexcorm
+  - stephanierobin
+  - r1corre
+  - lleroi 
+level: Introductory 
+---
+
+# Steps before starting a genome project
+
+.left[
+- Step 1: Build a wide community for the project if possible
+
+- Step 2: Gather information about the target genome
+
+- Step 3: Select the best possible DNA source and extraction
+
+- Step 5: Choose an appropriate sequencing technology
+
+- Step 6: Check the computational resources and requirements
+]
+
+---
+
+### Build a wide community for the project (*if it's possible*)
+
+.left[
+The aim of a genome projects is to **sequence the entire target genome** for a wide **range of genomics applications**.
+]
+
+.left[
+**Analyses**, **reanalyses** and **integration** of genomic and other phenotype information require:
+]
+
+- **Facilities:** Wet lab, sequencing, bioinformatics,…
+- **Personnel:** Skill intensive
+- **Software:** Knowledge intensive
+
+.left[
+/!\ Data storage, maintenance, transfer, and analysis costs will also likely remain substantial and represent an increasing proportion of overall sequencing costs in the future.
```

Comment:
is this intended to be a warning symbol or so? If so this works:

```suggestion
{% icon warning %} Data storage, maintenance, transfer, and analysis costs will also likely remain substantial and represent an increasing proportion of overall sequencing costs in the future.
```

Reply:
yes it is! Thank you!